UefiPayloadPkg/UefiPayloadPkgIa32.dsc

```diff
@@ -505,6 +505,12 @@
   OvmfPkg/EnrollDefaultKeys/EnrollDefaultKeys.inf
 !endif
 
+!include DasharoModulePkg/DasharoModuleComponents.dsc.inc
+!include DasharoModulePkg/Include/UserAuthFeature.dsc

```

Comment:
One of the DSC fiel has component.x64 definition. How it is supposed to work in IA32? Remove all modifications from this file as we do not use 32bit payload anyways.